modules/openshift-cluster-maximums.adoc

```diff
@@ -25,7 +25,7 @@
 | 250
 | 250
 | 250
-| 250
+| 500
```

Comment:
A few notes:
1) This was only tested to 100 nodes.
2) The default maxPods hasn't changed (still 250).

@jtaleric @ashishkamra thoughts on how to express this here (or do we even need to)?

Reply:
There are numbered notes at the end of the cluster maximums table. I think it will be ok to mention these notes there. Some suggested edits:
1. This was tested on a cluster with 100 worker nodes with 500 pods per worker node. The default maxPods is still 250. To get to 500 maxPods, the cluster needs to be created with a hostPrefix of 22 in the install-config.yaml file and maxPods set to 500 via a custom KubeletConfig.